java/ql/src/semmle/code/java/security/ControlledString.qll

```diff
@@ -10,9 +10,8 @@ import semmle.code.java.security.Validation
 /**
  * Holds if `method` is a `toString()` method on a boxed type. These never return special characters.
  */
-private predicate boxedToString(Method method) {
-  method.getDeclaringType() instanceof BoxedType and
-  method.getName() = "toString"
+private predicate boxedToString(ToStringMethod method) {
+  method.getDeclaringType() instanceof BoxedType
```

Comment:
This change should be reverted, as it also needs to cover the other `toString(..)` overloads on boxed types.

Reply:
(Caught by the failing tests.)

Reply:
Sorry; reverted the change and modified the QLDoc to mention this.